spec/controllers/idv/phone_errors_controller_spec.rb

```diff
@@ -109,20 +116,6 @@
     }
   end
 
-  before do
-    allow(idv_session).to receive(:user_phone_confirmation).
-      and_return(idv_session_user_phone_confirmation)
-    allow(idv_session).to receive(:current_user).and_return(user)
-    allow(idv_session).to receive(:previous_phone_step_params).
-      and_return(previous_phone_step_params)
-    allow(subject).to receive(:remaining_attempts).and_return(5)
-    allow(controller).to receive(:idv_session).and_return(idv_session)
-    stub_sign_in(user) if user
-
-    stub_analytics
-    allow(@analytics).to receive(:track_event)
-  end
-
```

Comment:
so much stubbing removed! 😍

Reply:
To be fair, the analytics and signin stubbing got moved up, but yeah, I don't know why idv_session was being stubbed.